### PR TITLE
linter: enable revive for planner/core/foreign_key.go

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: planner/core/foreign_key.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (f *FKCheck) AccessObject() AccessObject {
 }
 
 // OperatorInfo implements dataAccesser interface.
-func (f *FKCheck) OperatorInfo(normalized bool) string {
+func (f *FKCheck) OperatorInfo(_ bool) string {
```
Comment (Defined2014) [inline planner/core/foreign_key.go:82]: How about remove '_' here?

Comment (hawkingrei) [reply planner/core/foreign_key.go]: Done

File: planner/core/foreign_key.go [hunk 2]
```diff
@@ -116,7 +116,7 @@ func (f *FKCascade) AccessObject() AccessObject {
 }
 
 // OperatorInfo implements dataAccesser interface.
-func (f *FKCascade) OperatorInfo(normalized bool) string {
+func (f *FKCascade) OperatorInfo(_ bool) string {
```
Comment (Defined2014) [inline planner/core/foreign_key.go:119]:
```suggestion
func (f *FKCascade) OperatorInfo(bool) string {
```

**Review by Defined2014 — APPROVED** [review]

**Pull request merged**